config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

```diff
@@ -1,5 +1,42 @@
 # this file should contain all periodic jobs that use the fejta-bot token
 periodics:
+- name: periodic-api-review-help
+  interval: 1h
+  decorate: true
```

Comment:
I don't actually know what `decorate: true` does... this was copied from other commenter configs... is this right?

Reply:
it enables the pod-utils which automated git checkout, log upload, etc. previously these were handled by an entrypoint script (`jenkins/bootstrap.py`) - new jobs should prefer this.

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md